src/components/Menu/Menu.mdx

```diff
@@ -0,0 +1,150 @@
+
+## Component Props
+
+<Canvas of={MenuStories.WithControls} />
+
+<ArgTypes of={MenuStories.WithControls} />
+
+## Accessibility
+
+When the menu is open, the menu items are keyboard accessible with the `ArrowDown` and `ArrowUp` keys, but not tabbable.
+The focus indicator is the gray background and bolded text.
+The `Enter` key, or a click, selects an item and closes the menu.
+The menu can also be closed with a click outside of its container or the `Escape` key.
+
+The menu itself requires a label to provide meaningful accessible text, even when only the vertical ellipsis is used instead of a visible label.
+
+Resources:
+
+- [MDN Menu accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role)
+
+## Menu Types
+
+There are four types: button, text button, icon (with container), and icon (without container). These types are toggled with `showLabel` and `showBorder`.
+Use the icon type when there is limited space such as a table or on mobile.
+
+<Canvas of={MenuStories.MenuTypes} />
+
+## Menu Content
+
+Menu items can be displayed with images, icons, or just as text. They can be grouped with headings, dividers, or both.
+
+The maximum height of the menu list container is `320px`, or roughly eight menu items. Past that height, it will scroll. The
+maximum width is `300px`. Past that width, menu items will wrap onto the next line. It is recommended that, like the menu button,
+menu item labels are kept short.
+
+The menu can be aligned to the left or right by its consuming app.
+
+Content is passed to the menu through `listItemsData`, and each menu item needs to be one of three types: Action, Group, or Divider.
+Each type expects different properties, as shown below:
+
+<Source
+  code={`
+const exampleListItemsData = [
+  {
+    type: "group",
+    id: "group-1",
+    label: "I'm group 1",
+    children: [
+      {
+        type: "action",
+        id: "item-title-1",
+        label: "I'm item 1",
+        onClick: () => {
+          console.log("Item Title 1 clicked");
+        },
+      },
+      {
+        type: "action",
+        id: "item-title-2",
+        label: "I'm item 2",
+        onClick: () => {
+          console.log("Item Title 2 clicked");
+        },
+      },
+    ],
+  },
+  { type: "divider", id: "divider-1" },
+  {
+        type: "action",
+        id: "item-title-3",
+        label: "I'm item 3",
+        onClick: () => {
+          console.log("Item Title 3 clicked");
+        }
+  },
+];
+
+...
+
+<Menu labelText={"Menu"} listItemsData={exampleListItemsData} />
+`} language="tsx" />
+
+All `ActionItem`s require an `onClick` function, but the `media` prop is optional. Media will display
+to the left of the menu text, and will render either an `Icon` or `Image`. The `media` prop requires a type (icon or image)
+and then either the name of the icon, or the source url and alt text of the image.
```

Comment:
Mention that it's the name of a *DS icon, just to be clear to use the DS' set of names. Perhaps link to the Icon doc page as well.